dashboard/dashboard.css

```diff
@@ -0,0 +1,141 @@
+.dashboardSection {
```

Comment:
Agree, tried to use as less CSS as possible, so mostly colors and font stuff, however, constructed the lay-out of the dashboard with custom heights, was struggling a bit with that, I think it can be optimized but I could not get it to work the first try with only Bootstrap. Will add a comment and possibly a thing to solve in future